what4/src/What4/Expr/Builder.hs

```diff
@@ -951,7 +951,7 @@ findExprToRemove exprs0 = go [] exprs0 Nothing
           go (AppPPExpr a:prev) exprs (Just (reverse prev, a, exprs))
 
 
-ppExpr' :: forall t tp s . Expr t tp -> PPExprOpts -> ST s ([Doc], PPExpr)
+ppExpr' :: forall t tp s ann. Expr t tp -> PPExprOpts -> ST s ([Doc ann], PPExpr ann)
```

Comment:
Along with the previous comment, an explicit annotation type will make it useable at rendering time.

Reply:
Same reasoning as before; this is code that generates `Doc`s containing no annotations, so we might as well keep them polymorphic for maximum flexibility.